src/main/java/Todo.java

```diff
@@ -0,0 +1,17 @@
+public class Todo extends Task {
+
+    /*CONSTRUCTOR*/
+
+    public Todo(String description) {
+        super(description);
+        type = "T";
```

Comment:
Can consider not to declare the type here and just printTaskNotif() function to print the type out